ui/src/copainput.tsx

```diff
@@ -111,8 +111,14 @@ export function CopaInput(props: any) {
   const handleSelectedImageChange = (event: any, newValue: string | null) => {
     props.setSelectedImage(newValue);
     if (newValue !== null) {
-      let str1 = newValue.split(':').join('.');
-      let str2 = str1.split("/").join('.');
+      const split = newValue.split(":");
+      const str1 = split.join('.');
+      const str2 = str1.split("/").join('.');
+      if (split.length === 1) {
```

Comment:
what do we need the str1 and str2 variables for?

Reply:
they were left over from when the json file had a custom name, i have removed them